packages/w3up-client/README.md

```diff
@@ -129,13 +129,8 @@ const account = await client.login('zaphod@beeblebrox.galaxy')
 If your account doesn't have a payment plan yet, you'll be prompted to select one after verifying your email. A payment plan is required to provision a space. You can use the following loop to wait until a payment plan is selected:
 
 ```js
-// wait for payment plan to be selected
-while (true) {
-  const res = await account.plan.get()
-  if (res.ok) break
-  console.log('Waiting for payment plan to be selected...')
-  await new Promise((resolve) => setTimeout(resolve, 1000))
-}
+// Wait for a payment plan with a 1-second polling interval and 60-second timeout
+await client.waitForPaymentPlan(account, 1000, 60000)
```

Comment:
🙏 can we land as a separate PR? Apologies - I should have asked...

I think it might be better for this to be an instance method of `AccountPlan` rather than the top level? WDYT?

```js
await account.plan.wait()
```

In the docs I think it's simplier to just use the defaults (i.e. don't specify the interval/timeout).